### ISSUE TITLE
Reverse orientation of cross-section

### ISSUE THREAD

**Issue by EvanBianco** [agile-geoscience/geotransect#9]
Perhaps in the config file, we could add an argument to change the orientation of the plot; i.e. switch east on the left, west on the right, to west on the left, east on the right. 

Alternately, annotate the map and cross-section with an A-A' -type markings

**Comment by kwinkunks** [comment]
I'd like both of these things... right now the plot does not 'match' the map. I'm not even sure what would determine the orientation we do see. Calling this a bug.

**Comment by EvanBianco** [comment]
Agreed, I'm not sure how it gets the orientation. It must have to do with the order of the points within the transect shapefile. Doing something other than a reverse command in the config file with require some cunning way of determining "normal" means for North-East-South-West, and what is "reverse". Maybe the we can classify the profile as belonging to one of four orientations: S-N, SW-NE, W-E, and NW-SE, and set those as "normal". Reverse could mean N-S, NE-SW, E-W, and SE-NW, respectively.

In the axes themselves, this could be done by reversing the "extents[0] and extents[1]"
i.e.
extents[0] = extents[1]
extents[1] = extents[0]

Lastly, Labeling with A-A' on both map and section would follow from "normal" and "reverse". In normal we would have A on left, A' on right. Reverse would have A' on left, A on right...etc.
